hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -134,7 +136,8 @@ public Response getClusterState() {
       totalKeys += fileRecord.getValue();
     }
     builder.setKeys(totalKeys);
-
+    // Subtract deleted containers from total containers.
+    containers = containers - deletedContainersCount;
```

Comment:
A unit test would be good to have for this.

Reply:
Will do just wanted to confirm if the approach was correct. !

Reply:
@sadanand48 I have added the UT's